vignettes/hazardItems.Rmd

```diff
@@ -53,6 +53,13 @@ install.packages("hazardItems",
     dependencies = TRUE)
 ```
 
+#Updating 
+To update **hazardItems**
+
+```{r, eval=FALSE}
+update.packages(repos = "http://owi.usgs.gov/R")
```

Comment:
this should maybe be repos = c("http://owi.usgs.gov/R", "http://cran.rstudio.com/") so that it will update other packages that aren't in GRAN.